### ISSUE TITLE
Prevent concurrent re-deployments

### ISSUE THREAD

**Issue by espositocloud** [figwasp/figwasp#1]
The README mentions to schedule the CronJob every minute, and this might cause a new Job instance to start before the previous one completes, which in turn means the same Deployment resource can be triggered once a re-deployment is already ongoing.

In order to prevent this, I'd to propose few options:

- set `concurrencyPolicy: Forbid` to prevent concurrent Jobs
- in addition to the above parameter, consider lowering the frequency
- as alternative, replace the CronJob with a Deployment and an internal loop

**Comment by joel-ling** [comment]
Thank you, @espositocloud, for your proposals.

Having more than one instance of Figwasp running simultaneously would unnecessarily flood the image repositories with requests, and could result in the updating of a Deployment already being updated, as you have pointed out.

Following your suggestion, I have added `concurrencyPolicy: Forbid` to the example CronJob spec in the [`README`](https://github.com/figwasp/figwasp#run-figwasp-as-a-cronjob). I have also replaced its `cron` schedule with a more realistic example, although the original was more of a placeholder than a prescription.

Thank you for your contribution to the project!
